x-pack/plugins/fleet/common/constants/global_data_tags_exclude.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const GLOBAL_DATA_TAG_EXCLUDED_INPUTS = new Set<string>([
```

Comment:
I think that these strings should be their own constants, and some of them are already defined in the code. For instance, `apm` is defined in https://github.com/elastic/kibana/blob/c99d8c2335bc2ec4f1ea59bd72a87827bf13cc0b/x-pack/plugins/fleet/common/constants/epm.ts#L18 as well as `cloud_defend`. You can probably add them directly in that file.

Reply:
As you suggested I moved all these strings to the epm file. Fleet server and cloud defend package names were not exactly in the format that I need. The package names have underscores (eg. fleet_server) but input types have dashes (eg. fleet-server). I opted for using the already existing package names instead of creating new ones with dashes instead of underscores to avoid any confusion.